docs/src/index.md

```diff
@@ -4,14 +4,51 @@
 CurrentModule = InfrastructureSystems
 ```
 
-**The Documentation is still under construction, some sections are unfinished. The available methods have autogenerated documentation in the internal API section**
+**Installation**
+The latest stable release of PowerSystems can be installed using the Julia package manager with
 
-## Overview
+```julia
+] add InfrastructureSystems
+```
+
+For the current development version, "checkout" this package with
 
+```julia
+] add InfrastructureSystems#master
+```
+
+**Overview**
 `InfrastructureSystems.jl` is a [`Julia`](http://www.julialang.org) package that utilities for
```

Comment:
package that provides data management services and common utility software for the packages in ...